### PR TITLE
OS SDK 3.5.0 API breaking changes

### PR DESCRIPTION
**Pull request by jmarrec**
I may not have caught all of them, but I did catch a few. Tried to maintain backward compatibility.

I tried to make the commits (and their messages) explicit, so it should be redundant to add a review as a walkthrough of my changes, please use each commit to review. Also, lots of whitespace changes (trailing spaces and some tab characters), so if viewing on github add `?w=1` to the url, you'll see it's actually 60 lines changed only: https://github.com/NREL/openstudio-standards/pull/1393/files?w=1

Found via https://github.com/NREL/openstudio-common-measures-gem/pull/121

### CODE REVIEW

File: lib/openstudio-standards/standards/Standards.Model.rb [hunk 1]
```diff
@@ -1885,9 +1885,9 @@ def model_get_baseline_system_type_by_zone(model, climate_zone, custom = nil)
           # separate the primary zones from the secondary zones.
           # Add the baseline system type to the primary zones
           # and add the suplemental system type to the secondary zones.
-          story_zone_lists.each do |zones|
+          story_zone_lists.each do |story_group|
             # Differentiate primary and secondary zones
-            pri_sec_zone_lists = model_differentiate_primary_secondary_thermal_zones(model, story_group, zone_fan_scheds)
+            pri_sec_zone_lists = model_differentiate_primary_secondary_thermal_zones(model, story_group)
```
Comment (jmarrec) [inline lib/openstudio-standards/standards/Standards.Model.rb:1890]: Unrelated undefined local variable found in openstudio-common-measures-gem `GenericQAQC_Test#test_GenericQAQC_pass` throws

```
undefined local variable or method `story_group' for #<ASHRAE9012013:0x0000561f90a41108>)
/home/julien/Software/Others/openstudio_gems/openstudio-standards/lib/openstudio-standards/standards/Standards.Model.rb:1890:in `block (2 levels) in model_get_baseline_system_type_by_zone'
```

Also

```
undefined local variable or method `zone_fan_scheds' for #<ASHRAE9012013:0x0000564cbad042a8>
```

It's been there for more than 2 years though? https://github.com/NREL/openstudio-standards/blob/91b54056f8dd345589079f1c24abad51d14e6d10/lib/openstudio-standards/standards/Standards.Model.rb#L2191-L2193

**Comment by tijcolem** [comment]
@jmarrec Thanks Julien!  I update the Jenkinsfile to test against the latest development version of OpenStudio.

**Comment by tijcolem** [comment]
This block here is causing many, many failures when testing against 3.5.0-alpha due to a deprecated method that has been removed. https://github.com/NREL/openstudio-standards/blob/3.5.0_changes/lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb#L1760-L1767

It just needs to be updated to use the replacement method - availabilityManagers

See: https://github.com/NREL/OpenStudio/blob/develop/developer/ruby/deprecated_methods.csv

**Review by mdahlhausen — APPROVED** [review]
Thanks for doing this!  I just made a few minor syntax changes.

**Pull request merged**